common/components/componentsBySection/FindProjects/ProjectCardsContainer.jsx

```diff
@@ -52,6 +60,35 @@ class ProjectCardsContainer extends React.Component<{||}, State> {
             />
         );
   }
+
+  _handleFetchNextPage(e: object): void {
+    e.preventDefault();
+    this.setState({current_page: this.state.current_page + 1 <= this.state.project_pages 
```

Comment:
This would look cleaner if the current_page logic lived in its own expression.

Reply:
ok